index.html

```diff
@@ -1,26 +1,113 @@
-<!DOCTYPE html> 
+<!DOCTYPE html>
 <html lang="en">
-  
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="device-width, initial-scale=1.0"> 
-    <link rel="stylesheet" href="styles.css"> 
-   </head>
-  
-  <body>
-    <header>Glazed Donuts</header> 
-    
-    <main> </main>
-    
-    
-    
-    
-    
-    
-    <!--https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut01.jpg?v=1602793076372 --> 
-    <!--https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut02.jpg?v=1602793077492 --> 
-    
-    
-  </body>
-  
-</html>
\ No newline at end of file
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fglazed%20donut01.jpg?v=1602793076372" alt="">
+
+  <img src="https://cdn.glitch.com/e9974854-a7d9-497c-a277-5936bf92fc45%2Fspiceracklogo.png?v=1603324422595" alt="">
+
+  <h1>glazed Donuts</h1>
```

Comment:
For the purposes of the card component, you should downgrade this to an `h2` -- we're assuming there's another `h1` somewhere on the page (that you do not have to design as part of this project), but is a heading for a _group_ of cards (for example, "Best Donut Recipes", "Great for Breakfast", etc.).